grammar.js

```diff
@@ -78,12 +78,8 @@ module.exports = grammar({
     // Simple statements
 
     _simple_statements: $ => seq(
-      $._simple_statement,
-      optional(repeat(seq(
-        $._semicolon,
-        $._simple_statement
-      ))),
-      optional($._semicolon),
```

Comment:
I think you can delete the `_semicolon` rule from the grammar now

Reply:
True! I thought I had already deleted it. I'll push more commits later.